### ISSUE TITLE
Make navbar outline color configurable

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#12605]
https://github.com/joomla/joomla-cms/pull/11724 added an outline for navbar elements for improved accessibility, however uses a hardcoded color code.  Optimally this should be configurable; not every navbar will be the same shade of blue as Isis.

Interestingly enough this presents a small issue.  To make this configurable it would require introducing a LESS variable.  Is doing so considered a B/C break?  For templates compiling using the core LESS (i.e. the `joomla.org` template) they would not be able to compile against this source until the new variable is added.

**Comment by brianteeman** [comment]
Unfortunately it would appear that the Pr has been overwritten and is not in staging

**Comment by mbabker** [comment]
Can't fix, requires a B/C break.
